task-manager-backend/src/task/controllers/get-all-tasks/get-all-tasks.js

```diff
@@ -0,0 +1,23 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { getAllTasksUsecase } = require("../../usecases");
+
+const { TASK_FETCH_SUCCESS } = require("../../../commons/constants");
+
+module.exports = {
+    getAllTasks: {
+        path: "/",
+        reqType: "get",
+        method: async (req, res, next) => {
+            try {
+                const userId = req.session.userId;
+                const response = await getAllTasksUsecase.execute(userId);
```

Comment:
this user case name should be getTasksByUserId

Reply:
okay